### ISSUE TITLE
Notes are overwritten upon "Auto-Tag" even with "Overwrite metadata" unchecked

### ISSUE THREAD

**Issue by ducky0518** [comictagger/comictagger#390]
Hello I'm using the current v1.5.4

I have a .cbz file that already has some fields filed in (like notes). I'm expecting for Comictagger to append (add) to the notes section with the banner and issueID, but it appears it completely overwrites whats in there even though the "Overwrite metadata" box is unchecked which comes up when you start the auto tagging process.

Should be relatively easy to reproduce. Let me know if any questions. Thanks!

**Comment by lordwelch** [comment]
The "Overwrite metadata" checkbox clears all metadata before applying metadata from ComicVIne, it might make sense to rename that particular option. Currently when metadata is applied it will copy any non-empty metadata, the notes section would always be set and so would always replace whatever already in it.

**Comment by ducky0518** [comment]
So, how do I keep what's in the notes section already without it being overwritten?

Also, you say "it will copy any non-empty metadata" - and do what with it?

**Comment by lordwelch** [comment]
> So, how do I keep what's in the notes section already without it being overwritten?

ComicTagger doesn't have a way to preserve it right now. I'm looking to see what other metadata might have a reason to be preserved, obviously with the notes being entirely generated by ComicTagger it makes sense to provide an option preserve any notes that already exist. I probably won't make it append to the notes as I would have to check if it was already there and that can quickly get out of hand, I will probably just make it so the notes is only set if it is empty.

> Also, you say "it will copy any non-empty metadata" - and do what with it?

In other words:
`comic1.cbz(notes='This is a note', title='this is a title') <- ComicVine(notes='<by ComicTagger>', title='')`
becomes 
`comic1.cbz(notes='<generated by ComicTagger>', title='this is a title')`

if you have the "Overwrite metadata" box checked:
`comic1.cbz(notes='This is a note', title='this is a title') <- ComicVine(notes='<by ComicTagger>', title='')`
becomes
`comic1.cbz(notes='<generated by ComicTagger>', title='')`

**Comment by ducky0518** [comment]
Basically let me explain. I have ComicRack setting things like an original filename and filepath and saving that to the notes section.
I then had been using the CV Scraper as part of ComicRack - but your image matching algorithm is awesome and I have more faith in that correctly matching. The issue, however, is that Mylar expects the IssueID to be at the very end of the notes section. (it looks for it in brackets at the end) - So if I want to keep my original filename info AND add have the issueID at the end - and ideally use ComicTagger to do the matching since it does it so well (it also does better scan info detection and capture). Does this make sense? Let me know your thoughts.

**Comment by lordwelch** [comment]
Definitely don't want to break mylar, I've never used it my self but it uses an old embedded version of ComicTagger.
I might be able to come up with something this weekend.

**Comment by ducky0518** [comment]
That would be very much appreciated. I'm willing to contribute funds if that would help! Let me know how I can get something to you, if that would help. Again, just looking for it to append the ComicTagger notes to the original notes that are already there. Thanks.

**Comment by lordwelch** [comment]
I just pushed som commits to address this, you can download a package from master [here] (scroll all the way to the bottom) to test.

[here]: https://github.com/comictagger/comictagger/actions/runs/3397733924

**Comment by ducky0518** [comment]
OMG - This worked - PERFECT! For real, I'm willing to pay for your efforts. This is BEYOND appreciated. Is there a patreon/paypal/bitcoin address, anything? It may not have been much effort for you but saves me literally hundreds of hours manually tagging
